doc/whats_new/v0.22.rst

```diff
@@ -280,6 +275,9 @@ Changelog
 - |Fix| :class:`impute.IterativeImputer` now works when there is only one feature.
   By :user:`Sergey Feldman <sergeyf>`.
 
+- |Efficiency| :meth:`ensemble.MissingIndicator.fit_transform` avoid repeated
```

Comment:
```suggestion
- |Efficiency| :meth:`impute.MissingIndicator.fit_transform` avoid repeated
```

Reply:
sorry, my bad again. I'll immediately make the changes.

Thanks.